_sass/blocks/_revenues_subpage_steps.scss

```diff
@@ -28,16 +28,27 @@ $onshore-renewables-secondary-color: $blue;
   @include respond-to(medium-up) {
     display: block;
     margin-top: 40%;
-    @include span-columns(3);
+    @include span-columns(2);
+    &[accordion-open=true] {
+      @include span-columns(3);
+    }
   }
+
 }
 
+
+.revenues_subpage-dyk_heading,
```

Comment:
Selector `revenues_subpage-dyk_heading` should be written in lowercase with hyphens<br>Selector `revenues_subpage-dyk_text` should be written in lowercase with hyphens